api/src/wfl/server.clj

```diff
@@ -89,14 +86,16 @@
   (letfn [(do-update! [{:keys [id uuid]}]
             (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
               (let [{:keys [watchers] :as workload}
-                    (workloads/load-workload-for-id tx id)]
+                    (workloads/load-workload-for-id tx id)
+                    ;; we don't support notifying emails yet
```

Comment:
This comment also confuses because `start-workload-manager` doesn't know about email.
All it knows is Slack notification.
Actually, we should consider dispatching the notifications generically, but not now.

Reply:
Yeah, I thought about making `all/deserialize-watcher` a multimethod, but didn't bother as we only support slack…